src/TurbulenceClosures/turbulence_closure_implementations/isopycnal_skew_symmetric_diffusivity.jl

```diff
@@ -116,36 +116,66 @@ struct FluxTapering{FT}
 end
 
 """
-    taper_factor_ccc(i, j, k, grid::AbstractGrid{FT}, buoyancy, tracers, tapering::FluxTapering) 
+    taper_factor(i, j, k, grid, closure, tracers, buoyancy) 
 
 Return the tapering factor `min(1, Sₘₐₓ² / slope²)`, where `slope² = slope_x² + slope_y²`
-that multiplies all components of the isopycnal slope tensor. All slopes involved in the
-tapering factor are computed at the cell centers.
+that multiplies all components of the isopycnal slope tensor. The tapering factor is calculated on all the
+faces involved in the isopycnal slope tensor calculation. The minimum value of tapering is selected.
 
 References
 ==========
 R. Gerdes, C. Koberle, and J. Willebrand. (1991), "The influence of numerical advection schemes
     on the results of ocean general circulation models", Clim. Dynamics, 5 (4), 211–226.
 """
-@inline function taper_factor_ccc(i, j, k, grid, buoyancy, tracers, tapering::FluxTapering)
-    # TODO: handle boundaries!
-    bx = ℑxᶜᵃᵃ(i, j, k, grid, ∂x_b, buoyancy, tracers)
-    by = ℑyᵃᶜᵃ(i, j, k, grid, ∂y_b, buoyancy, tracers)
-    bz = ℑzᵃᵃᶜ(i, j, k, grid, ∂z_b, buoyancy, tracers)
+@inline function tapering_factor(i, j, k, grid, closure, tracers, buoyancy)
 
-    slope_x = - bx / bz
-    slope_y = - by / bz
-    slope² = ifelse(bz <= 0, zero(grid), slope_x^2 + slope_y^2)
+    ϵᶠᶜᶜ = tapering_factorᶠᶜᶜ(i, j, k, grid, closure, tracers, buoyancy)
+    ϵᶜᶠᶜ = tapering_factorᶜᶠᶜ(i, j, k, grid, closure, tracers, buoyancy)
+    ϵᶜᶜᶠ = tapering_factorᶜᶜᶠ(i, j, k, grid, closure, tracers, buoyancy)
```

Comment:
What's the difference between "tapering factor" and "taper factor"? Is this just a typo?